### PR TITLE
Make thermal related test work on both 201911 and master

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
### Description of PR

Summary:
Make thermal related test work on both 201911 and master

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?

Now thermalctld works different between 201911 and master. For example, ```show platform fan``` displays 6 columns on 201911 but 8 columns on master. This PR makes it work both on master and 201911.

#### How did you do it?

1. Add a new property kernel version in SonicHost, for 201911, the version should be 4.9.0, for master, the version should be 4.19.0 or higher.
2. Make some changes in test cases according to the kernel version

#### How did you verify/test it?

Run thermal related regression test cases

#### Any platform specific information?

N/A

### CODE REVIEW

**Comment by Junchao-Mellanox** [comment]
@keboliu please help review, thanks!

File: tests/common/devices.py [hunk 1]
```diff
@@ -326,6 +337,13 @@ def _get_os_version(self):
         output = self.command("sonic-cfggen -y /etc/sonic/sonic_version.yml -v build_version")
         return output["stdout_lines"][0].strip()
 
+    def _get_kernel_version(self):
+        """
+        Gets the SONiC kernel version
+        :return:
+        """
+        return self.setup()['ansible_facts']['ansible_kernel'].split('-')[0]
```
Comment (wangxin) [inline tests/common/devices.py:345]: Can you consider using command like `uname -r` to get kernel version? The ansible setup command is a little bit too heavy.

File: tests/platform_tests/mellanox/mellanox_thermal_control_test_helper.py [hunk 2]
```diff
@@ -384,7 +403,7 @@ class FanData:
     PWM_MAX = 255
 
     # Speed tolerance
-    SPEED_TOLERANCE = 0.2
+    SPEED_TOLERANCE = 0.5
```
Comment (wangxin) [inline tests/platform_tests/mellanox/mellanox_thermal_control_test_helper.py:406]: Why increase speed tolerance?

Comment (Junchao-Mellanox) [reply tests/platform_tests/mellanox/mellanox_thermal_control_test_helper.py:406]: The speed tolerance in production code is 50%, here is to align the test case with it.

**Review by wangxin — APPROVED** [review]

**Pull request merged**